index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header class="header">
+    <div class="header-content">
+      <div class="logo">
+        <a href="#">My Logo</a>
+      </div>
+      <img class="hamburger" src="images/menu.svg" alt="menu">
+    </div>
+  </header>
+  <section class="section-text">
+    <div class="section-title"><h1>I'm Daniel <br>Glad to see <br>you!</h1></div>
+    <div class="section-content">I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
+  </section>
+  <footer>
+    <div class="footer-content">LET'S CONNECT</div>
+    <div class="main-footer-network-content">
+      <div class="footer-network twitter"><img src="Images/twitter.svg" alt="twitter"></div>
+      <div class="footer-network linkedin"><img src="Images/linkedin.svg" alt="linkedin"> </div>
+      <div class="footer-network M"> <img src="Images/m.svg" alt="M"> </div>
```

Comment:
- [ ] please try to make your project look almost exactly like that of the figma. that is you follow the fonts, colors, margins etc. align your lest connect text and icons well. 
- your design 👇🏾 
![Screenshot (46)_LI](https://user-images.githubusercontent.com/65298048/170428756-c54f7987-25c0-4b87-8a49-63ae46d5ba8d.jpg)
- figma design [here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) 👇🏾 
![Screenshot (39)](https://user-images.githubusercontent.com/65298048/170428923-dd3f8143-ac69-4c2c-88b0-34dbc83626d2.png)